tests/rptest/tests/rpk_acl_test.py

```diff
@@ -42,6 +44,43 @@ def _superclient(self):
                        password=self.superuser.password,
                        sasl_mechanism=self.mechanism)
 
+    @cluster(num_nodes=3)
+    def test_modify_then_query_error(self):
+        """
+        This test ensures that even in cases where multiple nodes may
+        think they are the leader, the freshest data is returned
+        """
+        superclient = self._superclient()
+        superclient.acl_create_allow_cluster(self.superuser.username,
+                                             "describe")
+
+        # Modify the election timeout settings
+        admin = Admin(self.redpanda)
+        admin.patch_cluster_config({'election_timeout_ms': 5000})
```

Comment:
does this work? the config property is actually raft_election_timeout_ms.

Reply:
Yes, confusingly:
```  
, raft_election_timeout_ms(
      *this,
      "election_timeout_ms",
      "Election timeout expressed in milliseconds",
      {.visibility = visibility::tunable},
      1'500ms)
```